client/packages/plugins/README.md

```diff
@@ -173,7 +173,7 @@ You can work on plugins as if they were part of the app (types should be shared,
 
 ```bash
 # From server directory
-cargo run --bin remote_server_cli -- generate-plugin-bundle -i ../client/packages/plugins/mynewplugin/frontend -o pluginbundle.json
+cargo run --bin remote_server_cli -- generate-plugin-bundle -i ../client/packages/plugins/myPluginBundle/frontend -o pluginbundle.json
```

Comment:
decided to change this since submodule example in readme uses myPluginBundle as plugin name